docs/source/pipeline_tutorial.md

```diff
@@ -223,21 +232,25 @@ Next, the following tasks are run on the single-exposure single-CCD images:
   via a call to `psfex_interp`. For MCCD, the modules `merge_starcat`, `mccd_plots`, and
   `mccd_interp` are called.
 
+The output directory for both the `mccd` and `psfex` options is `run_sp_tile_Sx_exp_SxSePsf`.
+This stores the output of SExtractor on the tiles (`tile_Sx`), on the exposures (`exp_Sx`),
+`setools` (`Se`), and the Psf model (`Psf`).  
+
 The following plots show an example of a single CCD, in the center of the focal plane.
 
 | Size-magnitude plot | Star magnitude histogram | Stars in CCD (mag) | Stars in CCD (size) |
 | --- | --- | --- | --- |
-| <img width="250" src="size_mag-2113737-10.png" title="Size-magnitude plot with star selection"> | <img width="250" src="hist_mag_stars-2113737-10.png" title="Magnitude histogram of selected stars"> | <img width="250" src="mag_star_field-2113737-10.png" title="Magnitude distribution in CCD"> | <img width="250" src="fwhm_field-2113737-10.png" title="Size distribution in CCD"> |
+| <img width="250" src="../source/images/size_mag-2113737-10.png" title="Size-magnitude plot with star selection"> | <img width="250" src="../source/images/hist_mag_stars-2113737-10.png" title="Magnitude histogram of selected stars"> | <img width="250" src="../source/images/mag_star_field-2113737-10.png" title="Magnitude distribution in CCD"> | <img width="250" src="../source/images/fwhm_field-2113737-10.png" title="Size distribution in CCD"> |
 | The stellar locus is well-defined | Magnitude distribution looks reasonable | Stars are relatively homogeneously distributed over the CCD | The uniform and small seeing of CFHT is evident |
 
 To contrast the last plot, here is the case of the CCD in the lower right corner, which shows a known (but yet unexplained) lack of stars
 in the lower parts:
 
-<img width="250" src="fwhm_field-2113737-35.png" title="Size distribution in CCD">
+<img width="250" src="../source/images/fwhm_field-2113737-35.png" title="Size distribution in CCD">
 
 The statistics output file for the center CCD #10:
 ```bash
-(shapepipe)  dap ~/ShapePipeRun $ cat output/shapepipe_run_2020-03-05_10-00-26/setools_runner/output/stat/star_stat-2113737-10.txt
+(shapepipe)  dap ~/ShapePipeRun $ cat star_stat-2113737-10.txt
```

Comment:
I don't think you need to include your system prompt here.

Reply:
Changed.